summingbird-storm/src/main/scala/com/twitter/summingbird/storm/StormPlatform.scala

```diff
@@ -194,14 +195,17 @@ abstract class Storm(options: Map[String, Options], transformConfig: Summingbird
       }
     }
 
-    val countersForSpout: Seq[(Group, Name)] = JobCounters.getCountersForJob(jobID).getOrElse(Nil)
     val isMergeableWithSource = getOrElse(stormDag, node, DEFAULT_FM_MERGEABLE_WITH_SOURCE).get
     val sourceParallelism = getOrElse(stormDag, node, parOpt.getOrElse(DEFAULT_SOURCE_PARALLELISM)).parHint
     val flatMapParallelism = getOrElse(stormDag, node, DEFAULT_FM_PARALLELISM).parHint
 
     if (isMergeableWithSource) {
       require(flatMapParallelism <= sourceParallelism, s"SourceParallelism ($sourceParallelism) must be at least as high as FlatMapParallelism ($flatMapParallelism) when merging flatMap with Source")
     }
+    val builder = BuildSummer(this, stormDag, node, jobID)
+    val lockedCounters = Externalizer(JobCounters.getCountersForJob(jobID).getOrElse(Nil))
```

Comment:
I don't think we need to lock here.  Also, I don't think this is doing anything because we're serializing the unboxed thing anyway.